build.gradle

```diff
@@ -672,6 +672,12 @@ project(':streams') {
     testCompile libs.junit
   }
 
+  configurations {
+    // manually exclude unnecessary zkclient transitive dependencies, remove this after KIP-4
+    compile.exclude module: 'jline'
+    compile.exclude module: 'netty'
+  }
+
```

Comment:
Can we instead add the exclusion to the zkclient dependency instead of globally?
```
compile (libs.zkclient) {
   exclude module: 'jline'
   exclude module: 'netty'
} 
```